python/nammu/controller/NammuController.py

```diff
@@ -107,6 +108,14 @@ def __init__(self):
         # Find windows shouldn't coexist
         self.finding = False
 
+        # Here are the current urls for nammu on github and the oracc docs
+        self.urls = {'nammu': 'https://github.com/oracc/nammu',
```

Comment:
I'd be more inclined to move this to WelcomeController. Member variables in Nammu controller should be more things that need to be shared between different controllers, or be strictly related to NammuController, and the url list is only relevant for the welcome window.